<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDateTimeDialog1080p</class>
 <widget class="QDialog" name="SetDateTimeDialog1080p">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1650</width>
    <height>899</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SetDateTimeDialog</string>
  </property>
  <widget class="QFrame" name="layoutFrame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>1641</width>
     <height>891</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>4</horstretch>
     <verstretch>4</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>872</width>
     <height>652</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBoxTimeZone">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>1541</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeFormatBorder">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelGpsSyncBorder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTimeFormat">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>751</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Format
24-Hour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGpsSync">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>751</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string> GPS Synchronization
On</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>1541</width>
      <height>131</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximumDate">
     <date>
      <year>2037</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2016</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeZoneBorder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>1581</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSystemTimeBorder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>1581</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>995</x>
      <y>730</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>1315</x>
      <y>730</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <zorder>labelSystemTimeBorder</zorder>
   <zorder>labelTimeZoneBorder</zorder>
   <zorder>labelGpsSyncBorder</zorder>
   <zorder>labelTimeFormatBorder</zorder>
   <zorder>comboBoxTimeZone</zorder>
   <zorder>buttonTimeFormat</zorder>
   <zorder>buttonGpsSync</zorder>
   <zorder>dateTimeEdit</zorder>
   <zorder>buttonSave</zorder>
   <zorder>buttonClose</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
